module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -870,6 +864,63 @@ protected function getDueDate($itemId, $showTime)
         return '';
     }
 
+    /**
+     * Should we use the legacy authentication mechanism?
+     *
+     * @return bool
+     */
+    protected function useLegacyAuthentication(): bool
+    {
+        return $this->config['API']['legacy_authentication'] ?? false;
+    }
+
+    /**
+     * Support method to perform a username/password login to Okapi.
+     *
+     * @param string $username The patron username
+     * @param string $password The patron password
+     *
+     * @return Response
+     */
+    protected function performOkapiUsernamePasswordAuthentication(string $username, string $password): Response
+    {
+        $newMethod = !($this->config['API']['legacy_authentication'] ?? false);
```

Comment:
This should use your `useLegacyAuthentication` function?

Reply:
This variable was not being used for anything -- I just forgot to delete it after I refactored the rest of the code. Thanks for catching that!